pkg/lint/manifest.go

```diff
@@ -111,16 +115,6 @@ func LintSection(manifestID string, section packaging.Section) []LintError {
 	if section.ID == "" {
 		lint("Section ID absent")
 	}
-	re := regexp.MustCompile(`^(\d+)\.(\d+)$`)
```

Comment:
Should the rule IDs be completely free form? Or should we restrict spaces and symbols? Just thinking we might want to use them, for example, in URLs in the Preflight Web UI.

Reply:
Can't OPA perform this validation? letters and underscores I think?

Reply:
This is the rule ID from the _policy manifest_, so it needs to match to a Rego rule so we can create the report. So we should validate it here to make sure it conforms to Rego's naming requirements otherwise it won't be able to match a Rego rule.

Reply:
Ahh right - makes more sense.

Reply:
@j-fuentes What do you think about enforcing Rego's rule name requirements for IDs? So the regex would be something like `^[_a-zA-Z][_a-zA-Z0-9]$` based on [Rego grammar](https://www.openpolicyagent.org/docs/latest/policy-reference/#grammar) for `var` names.

Reply:
Yeah, that makes sense.